pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -702,7 +702,9 @@ def append_tk_command_to_qiskit(
     if optype == OpType.CnY:
         return qcirc.append(qiskit_gates.YGate().control(len(qargs) - 1), qargs)
     if optype == OpType.CnZ:
-        return qcirc.append(qiskit_gates.ZGate().control(len(qargs) - 1), qargs)
+        new_gate = qiskit_gates.ZGate().control(len(qargs) - 1)
+        new_gate.name = "mcz"
```

Comment:
Can you freely choose this name? Could we do `cnz`?

Reply:
Unfortunately 'cnz' doesn't work because AerSimulater accepts the name 'mcz'.
In general, AerSimulater accepts the gete list on blow
https://github.com/CQCL/pytket-qiskit/issues/368#issuecomment-2238328473